data/projects.json

```
@@ -238,5 +238,14 @@
         "level": "intermediate",
         "site": "",
         "tags": "html, css, javascript, node.js, mongodb"
+    }, {
```

Please add a new line after comma
```suggestion
    },
    {
```